CMakeLists.txt

```diff
@@ -12,20 +14,25 @@ endif()
 
 find_package(ament_cmake REQUIRED)
 find_package(cv_bridge REQUIRED)
-find_package(darknet_vendor REQUIRED)
+find_package(Darknet REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(rclcpp_components REQUIRED)
 find_package(sensor_msgs REQUIRED)
 find_package(vision_msgs REQUIRED)
 
+include_directories(
+  ${Darknet_INCLUDE_DIR}
```

Comment:
Please delete this `include_directories()` call.

 `Darknet::dark` appears to be a modern CMake target, and those usually include the include directories  via the target's `INTERFACE_INCLUDE_DIRECTORIES` property. It looks like the [`Darknet_INCLUDE_DIR` is taken from the target](https://github.com/AlexeyAB/darknet/blob/d2c3788539bb73d3cfcdb7a1d5fa6b29ae1ec0e1/DarknetConfig.cmake.in#L43-L45), so the variable adds nothing new.

Reply:
Clearly, seems that I forgot to remove from a prior version I was using.

Reply:
Clearly, seems that I forgot to remove from a prior version I was experimenting with.